proposals/0000-initial-extension-categorization.md

```diff
@@ -0,0 +1,224 @@
+|------------------------|----------------|-------------------------|
+| ApplicativeDo          | Experimental   | Stable                  |
+| Arrows                 | Experimental   | Stable                  |
+| GHCForeignImportPrim   | Experimental   | Stable                  |
+| ExtendedLiterals       | Experimental   | Stable                  |
+| ImpredicativeTypes     | Experimental   | Stable                  |
+| JavaScriptFFI          | Experimental   | Stable                  |
+| LinearTypes            | Experimental   | Stable                  |
+| OverloadedRecordUpdate | Experimental   | Stable                  |
+| RequiredTypeArguments  | Experimental   | Stable                  |
+
+### 2.2.3 Deprecated Extensions
+
+The following are extensions that are to be categorized as `Deprecated` when enabled, and for the `No` version are categorized as `Stable`.
+
+| Extension                         | Categorization | Negative Categorization |
+|-----------------------------------|----------------|-------------------------|
+| AlternativeLayoutRule             | Deprecated     | Stable                  |
+| AlternativeLayoutRuleTransitional | Deprecated     | Stable                  |
+| DoRec                             | Deprecated     | Stable                  |
+| NullaryTypeClasses                | Deprecated     | Stable                  |
+| OverlappingInstances              | Deprecated     | Stable                  |
+| ParallelArrays                    | Deprecated     | Stable                  |
+| PolymorphicComponents             | Deprecated     | Stable                  |
+| Rank2Types                        | Deprecated     | Stable                  |
+| RecordPuns                        | Deprecated     | Stable                  |
+| TypeInType                        | Deprecated     | Stable                  |
+
+## 2.3 Uncategorized Extensions
+
+The following are the extensions that are uncategorized as of this proposal. They are left uncategorized to allow progress overall on categorization by reducing the amount of discussion on any one proposal. They will be categorized with further discussion in some following proposal(s).
+
+| Extension               |
+|-------------------------|
+| AllowAmbiguousTypes     |
+| AutoDeriveTypeable      |
+| DoAndIfThenElse         |
+| DuplicateRecordFields   |
+| FieldSelectors          |
+| FunctionalDependencies  |
+| IncoherentInstances     |
+| LexicalNegation         |
+| ListTuplePuns           |
+| NegativeLiterals        |
+| PartialTypeSignatures   |
+| PatternSignatures       |
+| QuantifiedConstraints   |
+| QuasiQuotes             |
+| RebindableSyntax        |
+| RelaxedLayout           |
+| RelaxedPolyRec          |
+| Safe                    |
+| ScopedTypeVariables     |
+| StaticPointers          |
+| Strict                  |
+| TemplateHaskell         |
+| TemplateHaskellQuotes   |
+| Trustworthy             |
+| TypeAbstractions        |
```

Comment:
Why `TypeAbstractions` are uncategorized when `RequiredTypeArguments` are experimental?

Reply:
In short, because I missed it. I'll add `TypeAbstractions`, the user's guide already says `Experimental` anyway.